### ISSUE TITLE
`phone_device` table isn't created when using `--nomigrations`

### ISSUE THREAD

**Issue by klistwan** [jazzband/django-two-factor-auth#519]
I have a 2.2 Django project with `django-otp==0.8.0`, `django-two-factor-auth[phonenumbers]==1.14.0`, and an app `two_factor_auth` that makes use of these two libraries.

When tests are run with migrations, they pass:

```zsh
$ ./manage.py test two_factor_auth
  Applying two_factor.0001_initial... OK
  Applying two_factor.0002_auto_20150110_0810... OK
  Applying two_factor.0003_auto_20150817_1733... OK
  Applying two_factor.0004_auto_20160205_1827... OK
  Applying two_factor.0005_auto_20160224_0450... OK
  Applying two_factor.0006_phonedevice_key_default... OK
  Applying two_factor.0007_auto_20201201_1019... OK
----------------------------------------------------------------------
Ran 13 tests in 5.549s

OK
```

The migrations that are run are the ones from here: https://github.com/jazzband/django-two-factor-auth/tree/master/two_factor/migrations

But when we run tests without migrations, the tests fail:

```zsh
$ ./manage.py test two_factor_auth --nomigrations
django.db.utils.OperationalError: no such table: two_factor_phonedevice
```

I went back through our commit history and can confirm this issue started appearing when we:
- upgraded `django-otp` from 0.7.5 to 0.8.0
-  upgraded `django-two-factor-auth` from 1.9.1. to 1.14.0
- added `two_factor.plugins.phonenumber` as an INSTALLED_APP
- added `django-two-factor-auth[phonenumbers]==1.14.0` as a dependency

## Expected Behavior
We would expect running `$ ./manage.py test two_factor_auth --nomigrations` to also have all tests pass.

## Current Behavior
When tests are run without migrations (as is done in CI and local environments because our project has hundreds of migrations), they fail.

```
$ ./manage.py test two_factor_auth --nomigrations

Traceback (most recent call last):
  File "/opt/miniconda3/lib/python3.8/site-packages/two_factor/views/profile.py", line 38, in get_context_data
    'default_device': default_device(self.request.user),
  File "/opt/miniconda3/lib/python3.8/site-packages/two_factor/utils.py", line 16, in default_device
    for device in devices_for_user(user):
  File "/opt/miniconda3/lib/python3.8/site-packages/django_otp/__init__.py", line 80, in devices_for_user
    for device in model.objects.devices_for_user(user, confirmed=confirmed):
  File "/opt/miniconda3/lib/python3.8/site-packages/django/db/models/query.py", line 274, in __iter__
    self._fetch_all()
  ...
  File "/opt/miniconda3/lib/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 383, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: two_factor_phonedevice
```

## Your Environment

* Python version: 3.8
* Django version: 2.2
* django-otp version: 0.8.0
* django-two-factor-auth version: 1.14.0

**Comment by claudep** [comment]
Can you also provide your INSTALLED_APPS setting, please?

**Comment by klistwan** [comment]
Sure thing:

INSTALLED_APPS = (
    "corsheaders",
    "anymail",
    "django_extensions",
    "django_nose",
    "django.contrib.contenttypes",
    "django.contrib.auth",
    "django.contrib.sites",
    "grappelli",
    "django.contrib.sessions",
    "django.contrib.messages", 
    "django.contrib.staticfiles",
    "graphene_django",
    "django.contrib.admin",
    "django.contrib.humanize",
    "magic",
    "raven.contrib.django.raven_compat",
    "django_celery_beat",
    "absolute",
    "guardian",
    "rest_framework",
    "rest_framework.authtoken",
    "rest_framework_swagger",
    "markdown_deux",
    "nested_admin",
    "auditlog",
    "hijack",
    "compat",
    "hijack_admin",
    "django_otp",
    "django_otp.plugins.otp_static",
    "django_otp.plugins.otp_totp",
    "two_factor",
    "two_factor_auth",
    "two_factor.plugins.phonenumber",
)

**Comment by claudep** [comment]
Where is the implementation of --nomigrations?

**Comment by klistwan** [comment]
Ah, it appears we prepend `test_without_migrations` to `INSTALLED_APPS` (so it ends up as the first element) when the project is run in CI/local environments.

**Comment by klistwan** [comment]
I've tried changing where `test_without_migrations` appears in `INSTALLED_APPS`, but it hasn't made a difference. I did find a potential solution where we could disable migrations for all apps except `two_factor` with the following:

```python
class NoMigrations:
    """Disable migrations for all apps except two_factor"""
    def __getitem__(self, item):
        return None

    def __contains__(self, item):
        return item != "two_factor"

MIGRATION_MODULES = NoMigrations()
```

This would also allow us to remove `django-test-without-migrations` from our dependencies.

**Comment by claudep** [comment]
This issue is probably due to the fact we didn't move migration files when we moved the phone capability into the plugins folder. I feared to break existing installations. However it might be doable if done with care, to be tried...

**Comment by mick88** [comment]
I am having the same issue without `django-test-without-migrations` librart. My project uses `MIGRATE=False` option built into Django: https://docs.djangoproject.com/en/3.2/ref/settings/#migrate
Tests fail with this setting.

Models can be moved between apps. A special migration is needed: https://realpython.com/move-django-model/#the-short-way-reference-the-new-django-model-to-the-old-table
I've executed this successfully in the past.

**Comment by claudep** [comment]
Was fixed by #527
